.github/workflows/ci.yaml

```diff
@@ -83,7 +83,8 @@ jobs:
       - name: Get current latest plugin version from the base branch
         id: get-current-latest-version
         run: |
-          echo "version=$(cat package-lock.json | jq -r '.version')" >> $GITHUB_OUTPUT
+          PLUGIN_PATH="plugins/${{ matrix.affected-plugin }}/package-lock.json" 
```

Comment:
As during the npm publish, the version published is the one in package.json, I'd suggest to compare that instead of the lock.

Reply:
But this one is the package.lock in master